src/core/io/writer/h5md/h5md_core.cpp

```diff
@@ -0,0 +1,707 @@
+                                                             &chunk_dims_1d_single));
+        this->dataspace_particles_atoms_force_step = h5xx::dataspace(
+                dims_1d_single, maxdims_single);
+        this->dataset_particles_atoms_force_step =
+                h5xx::dataset(this->group_particles_atoms_force,
+                              "step", this->type_int,
+                              this->dataspace_particles_atoms_force_step,
+                              h5xx::policy::storage::chunked(1,
+                                                             &chunk_dims_1d_single));
+        /* particles -- atoms -- image */
+        this->group_particles_atoms_image =
+                h5xx::group(this->group_particles_atoms, "image");
+        this->dataspace_particles_atoms_image_value = h5xx::dataspace(
+                dims_3d, maxdims);
+        this->dataset_particles_atoms_image_value = h5xx::dataset(
+                this->group_particles_atoms_image,
+                "value", this->type_int,
+                this->dataspace_particles_atoms_image_value,
+                h5xx::policy::storage::chunked(3, chunk_dims_3d.data()));
+        this->dataspace_particles_atoms_image_value = h5xx::dataspace(
+                dims_3d, maxdims);
+        this->dataspace_particles_atoms_image_time = h5xx::dataspace(
+                dims_1d_single, maxdims_single);
+        this->dataset_particles_atoms_image_time =
+                h5xx::dataset(this->group_particles_atoms_image,
+                              "time", this->type_double,
+                              this->dataspace_particles_atoms_image_time,
+                              h5xx::policy::storage::chunked(1,
+                                                             &chunk_dims_1d_single));
+        this->dataspace_particles_atoms_image_step = h5xx::dataspace(
+                dims_1d_single, maxdims_single);
+        this->dataset_particles_atoms_image_step =
+                h5xx::dataset(this->group_particles_atoms_image,
+                              "step", this->type_int,
+                              this->dataspace_particles_atoms_image_step,
+                              h5xx::policy::storage::chunked(1,
+                                                             &chunk_dims_1d_single));
+
+        /* particles -- atoms -- species */
+        this->dataspace_particles_atoms_species = h5xx::dataspace
+                (chunk_dims_1d);
+        this->dataset_particles_atoms_species =
+                h5xx::create_dataset(this->group_particles_atoms, "species",
+                                     this->type_int, this->dataspace_particles_atoms_species);
+        this->WriteSpecies();
+        /* particles -- atoms -- parameters */
+        this->group_parameters = h5xx::group(this->h5md_file, "parameters");
+        this->group_parameters_vmd_structure = h5xx::group(
+                this->group_parameters, "vmd_structure");
```

Comment:
What's in the `/parameters/vmd_structure` group? There shouldn't be anything VMD-specific in these files as H5MD should be readable by all kinds of visualization programs.